corehq/apps/data_interfaces/models.py

```diff
@@ -141,8 +141,20 @@ class AutomaticUpdateRuleCriteria(models.Model):
     class Meta:
         app_label = "data_interfaces"
 
+    def _get_case_value(self, case, property_name):
+        if property_name.lower().startswith('parent/'):
```

Comment:
i just did a quick search and didn't find anything - the thing that comes closest is https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/xform.py#L1709, but it doesn't operate on cases. i agree this is a reusable piece of code, i'll move it elsewhere